app/kubernetes-ingress-controller/reference/configuration-options.md

```diff
@@ -99,6 +100,10 @@ rows:
     type: '`string`'
     description: The cluster domain. This is used e.g. in generating addresses for upstream services.
     default: ""
+  - flag: '`--combined-services-from-different-httproutes`'
+    type: '`bool`'
+    description: Combine rules from different HTTPRoutes that are sharing the same combination of backends to one Kong service to reduce total number of Kong services.
+    default: '`false`'
```

Comment:
Can we add something to the generator for this page to tag each item with the version it was generated for? That looks something like this:
```suggestion
  - flag: '`--combined-services-from-different-httproutes` {% new_in 3.5 %}'
    type: '`bool`'
    description: Combine rules from different HTTPRoutes that are sharing the same combination of backends to one Kong service to reduce total number of Kong services.
    default: '`false`'
```